src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.cpp

```diff
@@ -187,62 +245,40 @@ bool AclDeconvExecutorBuilder::customIsSupported(const DeconvAttrs &deconvAttrs,
+    auto dstTensorInfo  = aclDeconvTensorInfo.dstTensorInfo;
+    auto deconv_info    = aclDeconvTensorInfo.deconv_info;
 
     // After stride=8 up-sampling in ACL Deconvolution layer slower than reference
-    if (deconv_info.stride().first >= 8 || deconv_info.stride().second >= 8) return false;
+    if (deconv_info.stride().first >= 8 || deconv_info.stride().second >= 8) {
+        DEBUG_LOG("AclDeconvExecutor does not support strides > 8:");
+        return false;
+    }
 
     unsigned int dilation_x = (deconvAttrs.dilation.size() > 1) ? deconvAttrs.dilation.at(1) : deconvAttrs.dilation.at(0);
     unsigned int dilation_y = deconvAttrs.dilation.at(0);
     if (!one_of(dilation_x, static_cast<unsigned int >(0), static_cast<unsigned int >(1)) ||
         !one_of(dilation_y, static_cast<unsigned int >(0), static_cast<unsigned int >(1))) return false;
 
-    size_t in_h = srcDescs[0]->hasLayoutType(LayoutType::ncsp) ? srcDescs[0]->getShape().getDims()[2] : srcDescs[0]->getShape().getDims()[1];
-    size_t in_w = srcDescs[0]->hasLayoutType(LayoutType::ncsp) ? srcDescs[0]->getShape().getDims()[3] : srcDescs[0]->getShape().getDims()[2];
-
-    // Validate function has bug (https://github.com/ARM-software/ComputeLibrary/issues/1061) with error exception.
-    // We copy deconvolution_output_dimensions function for get correct validation
-    // TODO: remove after fix
-    if (validate_deconvolution_output_dimensions(in_w, in_h, kernel_x, kernel_y, deconv_info)) {
-        DEBUG_LOG("NEDeconvolutionLayer arm_compute::deconvolution_output_dimensions failed");
-        return false;
-    }
-
-    arm_compute::Status status = arm_compute::NEDeconvolutionLayer::validate(&srcTensorInfo,
-                                                                             &weiTensorInfo,
-                                                                             deconvAttrs.withBiasesParam ? &biasTensorInfo : nullptr,
-                                                                             &dstTensorInfo,
-                                                                             deconv_info);
-    if (!status) {
-        DEBUG_LOG("NEDeconvolutionLayer validation failed: ", status.error_description());
+    try {
+        arm_compute::Status status = arm_compute::NEDeconvolutionLayer::validate(&srcTensorInfo,
+                                                                                 &weiTensorInfo,
+                                                                                 deconvAttrs.withBiasesParam ? &biasTensorInfo : nullptr,
+                                                                                 &dstTensorInfo,
+                                                                                 deconv_info);
+        if (!status) {
+            DEBUG_LOG("NEDeconvolutionLayer validation failed: ", status.error_description());
+            return false;
+        }
+    } catch (...) {
```

Comment:
lets add a comment with a reference to ACL or OV issue about uncaught exception
so, we don't forget to remove this try-catch as soon as the issue is fixed

Reply:
@alvoron added comment